bin/local.js

```diff
@@ -68,7 +69,10 @@ function onFileChange(evt, filepath) {
 }
 
 function startFileWatch() {
-    watcher.watch(config.sourceLocation, {
+    if (fswatcher){
+        fswatcher.close();
```

Comment:
Yeah, I just wasn't familiar with that library and wanted to be sure the listeners were setup again after a disconnect.  It is probably safe to assume they are still there.